### PR TITLE
[80538] Shorthand syntax for array destruction

### PR DESCRIPTION
**Pull request by deleugpn**
This pull request updates the return value documentation for functions to reflect newer syntax provided by PHP as reported in https://bugs.php.net/bug.php?id=80538

### CODE REVIEW

File: language/functions.xml [hunk 1]
```diff
@@ -739,9 +739,13 @@ echo square(4);   // outputs '16'.
 <?php
 function small_numbers()
 {
-    return array (0, 1, 2);
+    return [0, 1, 2];
 }
-list ($zero, $one, $two) = small_numbers();
+[$zero, $one, $two] = small_numbers();
+
+// For PHP 7.0 or lower
```
Comment (Girgias) [inline language/functions.xml:746]: This comment doesn't make much sense, we don't document PHP 5 any more, so something more along the way of this being a "functional" (by using a function-style construct) way is probably better.

Comment (cmb69) [reply language/functions.xml]: Array destructoring is only available as of PHP 7.1.0, though, so it makes sense to mention that here. :)

Comment (deleugpn) [reply language/functions.xml]: That was precisely my intention - to mention an alternative for versions that don't have Array destruction yet.

Comment (cmb69) [reply language/functions.xml]: The usual wording in the manual is `prior to PHP 7.1.0` (or `as of PHP 7.1.0`), though.

File: language/functions.xml [hunk 2]
```diff
@@ -739,9 +739,14 @@ echo square(4);   // outputs '16'.
 <?php
 function small_numbers()
 {
-    return array (0, 1, 2);
+    return [0, 1, 2];
 }
-list ($zero, $one, $two) = small_numbers();
+// Array destruction will collect each member of the array individually
```
Comment (kamil-tekiela) [inline language/functions.xml:744]:
```suggestion
// Array destructuring will collect each member of the array individually
```

**Review by kamil-tekiela — CHANGES_REQUESTED** [review]

**Review by kamil-tekiela — APPROVED** [review]
LGTM

**Pull request merged**